lib/tasks/interactive-new.js

```diff
@@ -121,6 +121,22 @@ class InteractiveNewTask extends Task {
           },
         ],
       },
+      {
+        name: 'emberData',
+        type: 'confirm',
+        message: 'Do you want to include ember-data?',
+        when: undefined === newCommandOptions.emberData,
```

Comment:
This will always be `false` since the `ember-data` flag has a default value.

Reply:
https://github.com/ember-cli/ember-cli/pull/10622